plaso/storage/identifiers.py

```diff
@@ -103,3 +105,38 @@ def CopyToString(self):
       return '{0:s}.{1:d}'.format(self.name, self.row_identifier)
 
     return None
+
+
+class RedisKeyIdentifier(containers_interface.AttributeContainerIdentifier):
+  """Redis key attribute container identifier.
+
+  The identifier is used to uniquely identify attribute containers. Where
+  for example an attribute container is stored as a JSON serialized data in
+  a Redis instance.
+
+  Attributes:
+    identifier (UUID): unique identifier of a container.
+  """
+
+  def __init__(self, identifier=None):
+    """"Initializes a Redis key identifier.
+
+    Args:
+      identifier (Optional[str]): hexadecimal representation of a UUID4 UUID.
```

Comment:
Add some information about when the value is None a random UUID (version 4) is generated.

Reply:
Also maybe change `UUID4 UUID` to `a random UUID (version 4)`